todo-app/src/AppTask.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export class AppTask extends React.Component{
+    render(){
+       let date = this.props.tasks.map((elem, i)=>{
+           return <div>
```

Comment:
When you run your app via `yarn start`, try opening the browser console. You'll see two warnings from React. Both of them have to do with creating children in arrays (your `map` functions).

When creating React components in a loop, it's extremely important to give the top-level JSX tag a `key` prop. This `key` prop's value must be unique to that element.

While using index isn't great practice, we can leave it for the time being.

Your main problem here is that you provided the `key` prop to a child _inside_ your returned JSX, so React has no idea it's there. It needs to be a prop of the _highest level_ JSX returned, so in this case on the `div`.